### ISSUE TITLE
No Convolution0

### ISSUE THREAD

**Issue by conggalam12** [starimeL/PytorchConverter#48]
Unknown layer type: Convolution0, known types: dict_keys(['data', 'Addmm', 'Threshold', 'ConvNd', 'MaxPool2d', 'AvgPool2d', 'Add', 'Cmax', 'BatchNorm', 'Concat', 'Dropout', 'UpsamplingBilinear2d', 'MulConstant', 'AddConstant', 'Softmax', 'Sigmoid', 'Tanh', 'ELU', 'LeakyReLU', 'PReLU', 'Slice', 'MultiCopy', 'Negate', 'Permute', 'View'])
